CONTRIBUTING.md

```diff
@@ -92,7 +92,7 @@ the [Software Carpentry introduction to git][link_swc_intro].
 On GitHub, You'll use [Markdown][markdown] to chat in issues and pull requests.
 You can think of Markdown as a few little symbols around your text that will allow GitHub
 to render the text with a little bit of formatting.
-For example you could write words as bold (`**bold**`), or in italics (`*italics*`),
+For example, you could write words as bold (`**bold**`), or in italics (`*italics*`),
 or as a [link][rick_roll] (`[link](https://https://youtu.be/dQw4w9WgXcQ)`) to another webpage.
```

Comment:
This one isn't displaying properly.

Reply:
My mistake. Ignore this comment.

Reply:
Actually, there are two `https://https://youtu...` - although it does not affect the functionality of the link, it is completely wrong and defeats the purpose of showing how to write links. Fixed in the latest commit.